spec/requests/topics_controller_spec.rb

```diff
@@ -1203,6 +1203,14 @@ def topic_user_post_timings_count(user, topic)
           expect(response.parsed_body['basic_topic']).to be_present
         end
 
+        it 'should trigger only the `topic_edited` event' do
+          expect_enqueued_with(job: :emit_web_hook_event, args: { event_name: "topic_edited" }) do
```

Comment:
I think this assertion doesn't fully cover what the description requires which is to ensure that only the `topic_edited` event job is enqueued. We need to test for a count or assert thta the `post_edited` event is not fired.

Reply:
Is this test a duplicate of https://github.com/discourse/discourse/pull/10590/files#diff-0fae1daa2e2270483f864ea4a3e859f7R1235 ?